Assembly-CSharp/Memoria/Battle/SFX/BattleActionThread.cs

```diff
@@ -327,7 +328,12 @@ public static List<BattleActionThread> LoadFromBtlSeq(BTL_SCENE scene, btlseq.bt
 					case 14: // Battle Text
 					case 33: // Battle Title
 						Byte messId = r.ReadByte();
-						mainThread.code.AddLast(new BattleActionCode("Message", "Text", (messId & 128) != 0 ? "[CastName]" : battleText(messPtr + messId), "Priority", (messId & 128) != 0 ? "1" : "4", "Title", (seq.wSeqCode == 33 || (messId & 128) != 0).ToString(), "Reflect", (seq.wSeqCode == 33).ToString()));
+						string text = battleText(messPtr + messId);
+						mainThread.code.AddLast(new BattleActionCode("Message", "Text", (messId & 128) != 0 ? "[CastName]" : text, "Priority", (messId & 128) != 0 ? "1" : "4", "Title", (seq.wSeqCode == 33 || (messId & 128) != 0).ToString(), "Reflect", (seq.wSeqCode == 33).ToString()));
```

Comment:
Consider making this method call more compact (currently it is 260 characters). I understand, this method is not well documented, but at least it will be easier to read in this format.

```suggestion

						string arg1 = (messId & 128) != 0 ? "[CastName]" : text;
						string arg2 = (messId & 128) != 0 ? "1" : "4";
						string arg3 = (seq.wSeqCode == 33 || (messId & 128) != 0).ToString();
						string arg4 = (seq.wSeqCode == 33).ToString();

						BattleActionCode actionCode = new BattleActionCode("Message", "Text", arg1, "Priority", arg2, "Title", arg3, "Reflect", arg4);
						mainThread.code.AddLast(actionCode);
```

Reply:
Nit: If Tirlititi is going to replace this stuff, you might as well help him to with that by making code cleaner. On the other hand, this line change doesn't make it worse.